docs/fuchsia.md

```diff
@@ -39,8 +39,8 @@ Run `syz-manager` with a config along the lines of:
 	"workdir": "/workdir.fuchsia",
 	"kernel_obj": "/fuchsia/out/build-zircon/build-x64",
 	"syzkaller": "/syzkaller",
-	"image": "/fuchsia/out/x64/out/build/images/fvm.blk",
-	"sshkey": "/fuchsia/out/x64/ssh-keys/id_ed25519",
+	"image": "/fuchsia/out/x64/obj/build/images/fvm.blk",
+	"sshkey": ".ssh/pkey",
```

Comment:
What does create this file? I don't see any previous step that creates this file? It's a relative path, but against what? What is expected to be the current dir?